ckanext/ozwillo_organization_api/plugin.py

```diff
@@ -354,24 +346,24 @@ def setup_dataset_dict(city_insee):
     
     return dataset_dict
 
+
 def get_name_from_siret(siret):
 
-    apiKey = '08d7efcd484c4d7ac925c0f4e3e6ca75'
-    secretKey = '3bedcd17968fa9f996a52c3357011e05'
+    apiKey = config.get('ckanext.ozwillo_organization_api.verifsiret_apikey', '08d7efcd484c4d7ac925c0f4e3e6ca75')
+    secretKey = config.get('ckanext.ozwillo_organization_api.verifsiret_secretkey', '3bedcd17968fa9f996a52c3357011e05')
```

Comment:
It's recommended (to say the least) to not have any key or secret in the source code. If they are mandatory, the plugin should fail at startup if none is provided.